### DISCUSSION TITLE
something wrong when dpgen init bulk

### DISCUSSION THREAD

**Discussion by Normadp** [deepmodeling/dpgen#1680] [Q&A]
During the process of dpgen init bulk, it stayed at stage 3 and did not proceed to stage 4. The err file also did not report any errors. The following is the relevant information about err and dpgen.log
![1733142009074](https://github.com/user-attachments/assets/2ad2303e-5d2e-4c13-8b56-872e99b021ca)
![1733142324805](https://github.com/user-attachments/assets/a08ce28f-343e-4424-9c1d-e117fc27d15d)

**Comment by RO-CR** [comment]
I get the same error.
After I restart the third and fourth steps, I can successfully obtain the result.
